packages/mobile/src/components/comments/RecentUserCommentsDrawer.tsx

```diff
@@ -91,62 +92,50 @@ const CommentItem = ({ comment }: { comment: CommentOrReply }) => {
-              <Text
-                style={{ flexShrink: 3 }}
-                variant='body'
-                size='s'
-                color='subdued'
-                ellipses
-                lineHeight='single'
-                numberOfLines={1}
-              >
-                {track.title}
-              </Text>
-              <Text
-                variant='body'
-                lineHeight='single'
-                size='s'
-                color='subdued'
-                flexShrink={0}
-              >
-                {messages.by}
-              </Text>
-              <Text
-                style={{ flexShrink: 1 }}
-                variant='body'
-                lineHeight='single'
-                size='s'
-                color='subdued'
-                ellipses
-                numberOfLines={1}
-              >
-                {artist.name}
-              </Text>
-            </Flex>
-            {/* Commenter name, badges, date */}
-            <Flex row gap='s' alignItems='center'>
-              <Flex row gap='xs' alignItems='center'>
+          <Flex flex={1}>
+            <Flex gap='xs'>
+              {/* Track / artist name */}
+              <Flex row w='100%' style={{ flexShrink: 1 }}>
                 <Text
+                  style={{ flexShrink: 3 }}
                   variant='body'
-                  size='m'
-                  strength='strong'
+                  size='s'
+                  color='subdued'
                   ellipses
+                  lineHeight='single'
                   numberOfLines={1}
                 >
-                  {commenter?.name}
+                  {track.title}
+                </Text>
+                <Text
+                  variant='body'
+                  lineHeight='single'
+                  size='s'
+                  color='subdued'
+                  flexShrink={0}
+                >
+                  {messages.by}
                 </Text>
-                <Flex row gap='2xs'>
-                  <UserBadgesV2 userId={userId} badgeSize='xs' />
-                </Flex>
+                <Text
+                  style={{ flexShrink: 1 }}
+                  variant='body'
+                  lineHeight='single'
+                  size='s'
+                  color='subdued'
+                  ellipses
+                  numberOfLines={1}
+                >
+                  {artist.name}
+                </Text>
+              </Flex>
+              {/* Commenter name, badges, date */}
+              <Flex row gap='s' alignItems='center'>
+                <UserLink strength='strong' userId={userId} />
+                <Timestamp time={dayjs.utc(comment.createdAt).toDate()} />
               </Flex>
-              <Timestamp time={dayjs.utc(comment.createdAt).toDate()} />
             </Flex>
-            {/* Comment text */}
             <CommentText
-              isEdited={false}
+              isEdited={isEdited}
```

Comment:
Oh interesting, I assumed the edited tag was more of a thing for threads.

Reply:
i think it's just to indicate that the user edited the comment, so i guess we stay consistent?